docs/api/class-reference.md

```diff
@@ -148,13 +153,23 @@ This object represents a generic Creative. It's used as a parent object for more
 - `iconViewTrackingURLTemplate: String|null`
 
 ## AdExtension<a name="ad-extension"></a>
+
 - `name: String|null`
 - `value: any`
 - `attributes: Object`
 - `children: Array<AdExtension>`
 - `isEmpty(): Boolean` Returns true when none of these attributes have a value
 
+## CreativeExtension<a name="creative-extension"></a>
```

Comment:
please update the docs to just have one `Extension` object

Reply:
Done